packages/camera/src/components/Controls/__tests__/AddDamageButton.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
```

Comment:
Use `import { render, screen } from '../../../testUtils/i18Test-utils'` in place of this import statement